torch/_dynamo/comptime.py

```diff
@@ -143,7 +142,7 @@ def force_static(self) -> None:
         """
         if isinstance(self.__variable, SymNodeVariable):
             self.__variable.evaluate_expr()
-        elif isinstance(self.__variable, ConstantVariable):
+        elif self.__variable.is_python_constant():
```

Comment:
Why is this method instead anyway instead of a free function? A free function would play better with type inference if it used typing_extension.TypeIs. For override behavior of subclasses I am guessing?